### PR TITLE
cicd: add release action

### PR DESCRIPTION
**Pull request by jsstevenson**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          path: dist/
+  publish-to-pypi:
+    name: >-
+      Publish Python distribution to PyPI
+    needs:
+      - build
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/thera-py
```
Comment (korikuzma) [inline .github/workflows/release.yaml:40]:
```suggestion
      url: https://pypi.org/p/wags_tails
```

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (jsstevenson) [reply .github/workflows/release.yaml:40]: no!!!

Comment (jsstevenson) [reply .github/workflows/release.yaml:40]: how do my eyes never work

Comment (korikuzma) [reply .github/workflows/release.yaml:40]: It's EOD

**Review by korikuzma — APPROVED** [review]

**Pull request merged**